Plutarch/Api/V1.hs

```diff
@@ -401,17 +454,23 @@ instance PIsDataRepr (PExtended a) where
 ---------- Tx/Address
 
 data PCredential (s :: S)
-  = PPubKeyCredential (Term s (PDataList '[PPubKeyHash]))
-  | PScriptCredential (Term s (PDataList '[PValidatorHash]))
+  = PPubKeyCredential (Term s (PDataRecord '["_0" ':= PPubKeyHash]))
+  | PScriptCredential (Term s (PDataRecord '["_0" ':= PValidatorHash]))
   deriving
+<<<<<<< HEAD
+    (PMatch, PIsData, PUnsafeLiftDecl Plutus.Credential)
+    via (PIsDataReprInstances PCredential Plutus.Credential)
+  deriving (PDataFields) via (DerivePDataFields PCredential)
+=======
     (PMatch, PIsData)
     via (PIsDataReprInstances PCredential)
+>>>>>>> origin/staging
 
 instance PIsDataRepr PCredential where
   type
     PIsDataReprRepr PCredential =
-      '[ '[PPubKeyHash]
-       , '[PValidatorHash]
+      '[ '["_0" ':= PPubKeyHash]
+       , '["_0" ':= PValidatorHash]
```

Comment:
this doesn't look right

Reply:
I think it's correct, though field naming for single item constructors could be different...

Reply:
So it's legal to use the same field name twice?

Reply:
yep, in the case of sum types you can't use `pletFields` directly, you have to `pmatch` first, so it's safe.